moveit_ros/trajectory_cache/test/test_trajectory_cache.py

```diff
@@ -0,0 +1,154 @@
+# Copyright 2023 Johnson & Johnson
```

Comment:
The test coverage is really nice, thank you!

Reply:
No problems! I might need to change up some tests as I iterate on this PR.